### ISSUE TITLE
Incompatibility with Vivecraft, causes a mirror view of your HMD to be shown under your eyes

### ISSUE THREAD

**Issue by SpencerMeow** [Drillgon200/Hbm-s-Nuclear-Tech-GIT#120]
Here is what it looks like. I'm not sure why it does this, but I've narrowed it down to this mod
![2021-07-22_13 02 28](https://user-images.githubusercontent.com/58450660/126713403-073728b7-44a8-44e3-8090-7d29596b2dda.png)

**Comment by Drillgon200** [comment]
There's a good chance this has something to do with the post processing stuff I do, so try turning off enableShaders2 in the config.

**Comment by dc7431** [comment]
Tried it myself. Disabling enableShaders2 fixes the problem with the mirror view, but vivecraft/HBM has other weird issues.
(Some weapons are shown as arrows)
![2021-08-29_02 26 33](https://user-images.githubusercontent.com/67658810/131234385-706ad4f1-6f4c-4fd7-beea-7573a19d530e.png)
(Some weapons are too big)
![2021-08-29_02 25 51](https://user-images.githubusercontent.com/67658810/131234410-3d174cd2-3b1f-438f-b683-a3525c302c5a.png)
Some weapons/items sit incorrectly in the hand
Some weapons/items wont show at all.
And the usage of some items wont work(using the interaction button wont shoot an projectile/interact with the world).

**Comment by Drillgon200** [comment]
Yeah given that I don't have a vive, I don't know how I'm supposed to fix any of this.

**Comment by TheOriginalGolem** [comment]
I've got a quest, once final exams are done, I'll try make a compatibility patch.
